src/utils/search.ts

```diff
@@ -116,7 +116,7 @@ export function applySearchFilters(
 
 export function countOptionalSearchFiltersUsed(filters: SearchFilters): number {
   let count = 0;
-  const { zip, miles, ...optionalFilters } = filters;
+  const { ...optionalFilters } = filters;
```

Comment:
same feedback as above - we need `optionalFilters` to represent the filters that are not zip or miles, since those are required filters

Reply:
Will apply this comment as well https://github.com/bloom-works/ownpath/pull/155#discussion_r1017164064